ui/main/src/app/modules/archives/archives.component.ts

```diff
@@ -115,6 +119,12 @@ export class ArchivesComponent implements OnDestroy, OnInit {
         ).subscribe(() => this.setDateFilterBounds());
     }
 
+    toggleCollapsibleUpdates() {
+        this.isCollapsibleUpdatesActivated = !this.isCollapsibleUpdatesActivated;
+        localStorage.setItem('opfab.isCollapsibleUpdatesActivated', String(this.isCollapsibleUpdatesActivated));
+        this.sendQuery(0);
```

Comment:
Do not send the query until user click on search to be coherent with other input field

Reply:
The checkbox is not part of the filters, I think it is more user-friendly, as the checkbox and the search button are a little bit apart, it can be "boring" to click the one then the other each time.
So I propose to not modify this point if you agree ?

Reply:
OK for me